helpers/entradaHelper/entrada.helper.go

```diff
@@ -277,6 +289,69 @@ func AprobarEntrada(entradaId int) (result map[string]interface{}, outputError m
 	return resultado, nil
 }
 
+func asignarPlacaActa(actaRecibidoId int) (elementos []*models.Elemento, outputError map[string]interface{}) {
+
+	defer func() {
+		if err := recover(); err != nil {
+			outputError = map[string]interface{}{
+				"funcion": "asignarPlacaActa - Unhandled Error!",
+				"err":     err,
+				"status":  "500",
+			}
+			panic(outputError)
+		}
+	}()
+
+	if detalleElementos, err := actaRecibido.GetElementos(actaRecibidoId); err != nil {
+		outputError = map[string]interface{}{
+			"funcion": "asignarPlacaActa - actaRecibido.GetElementos(actaRecibidoId)",
+			"err":     err,
+			"status":  "502",
+		}
+		return nil, outputError
+	} else {
+		for _, elemento := range detalleElementos {
+			if elemento.SubgrupoCatalogoId.TipoBienId.NecesitaPlaca == true {
+				if placa, err := utilsHelper.GetConsecutivo("%05.0f", 8524, "Registro Placa Arka"); err != nil {
+					logs.Error(err)
+					outputError = map[string]interface{}{
+						"funcion": "asignarPlacaActa - utilsHelper.GetConsecutivo(\"%05.0f\", 8524, \"Registro Placa Arka\")",
+						"err":     err,
+						"status":  "502",
+					}
+					return nil, outputError
+				} else {
+					year, month, day := time.Now().Date()
+					elemento.Placa = strconv.Itoa(year) + fmt.Sprintf("%02d", int(month)) + fmt.Sprintf("%02d", day) + placa
```

Comment:
`printf` es de argumento variable, n+1 (el formato + las variables que consuma el formato), lo siguiente se puede reescribir como:
```suggestion
					elemento.Placa = fmt.Sprintf("%d%02d%02d%s", year, month, day, placa)
```
(Validar que lo anterior funciona igual, la idea es que se puede reducir)